app/views/npq_separation/admin/finance/statements/show.html.erb

```diff
@@ -1,5 +1,25 @@
 <%= tag.h1("#{@statement.lead_provider.name}, #{statement_name(@statement)}", class: "govuk-heading-l") %>
 
+<% content_for :side_navigation do %>
+  <hr class="govuk-section-break govuk-section-break--m">
+
+  <%= render NpqSeparation::SubNavigationComponent.new(
+    request.path,
+    heading: {
+      text: "Finance statement",
+      visible: true
+    },
+    structure: [
+      NpqSeparation::NavigationStructure::Node.new(
+        name: "#{@statement.lead_provider.name}, #{statement_name(@statement)}",
+        href: npq_separation_admin_finance_statement_path(@statement),
+        prefix: ""
```

Comment:
I actually have a change to the navigation structure component in [one of my PRs](https://github.com/DFE-Digital/npq-registration/pull/2592/files#diff-c7ee329aa04be00dd4737fabd61856919a50209c152fe155d4b22e7fd3e5b066) that will allow `prefix` to be omitted.
If prefix is `""`, then that side nav item is always highlighted, when it's omitted, then it's not highlighted. I'm not sure which is correct here.